src/shared/install.c

```diff
@@ -2521,8 +2517,8 @@ int unit_file_add_dependency(
                 size_t *n_changes) {
 
         _cleanup_(lookup_paths_free) LookupPaths lp = {};
-        _cleanup_(install_context_done) InstallContext c = {};
-        UnitFileInstallInfo *i, *target_info;
+        _cleanup_(install_context_done) InstallContext ctx = { .scope = scope };
```

Comment:
why not add line breaks after the { and before the }?

Reply:
I think that as long as only one field is initialized, it's OK to do it in one line. But I don't feel very strongly about this, I can change it if you think it looks better.